internal/network/command_access_point_private_link_egress_endpoint_create.go

```diff
@@ -22,12 +22,17 @@ func (c *accessPointCommand) newCreateCommand() *cobra.Command {
 				Text: "Create an AWS private link egress endpoint with high availability.",
 				Code: "confluent network access-point private-link egress-endpoint create --cloud aws --gateway gw-123456 --service com.amazonaws.vpce.us-west-2.vpce-svc-00000000000000000 --high-availability",
 			},
+			examples.Example{
+				Text: "Create a named Azure private link egress endpoint.",
+				Code: "confluent network access-point private-link egress-endpoint create my-egress-endpoint --cloud azure --gateway gw-123456 --service /subscriptions/0000000/resourceGroups/plsRgName/providers/Microsoft.Network/privateLinkServices/privateLinkServiceName",
+			},
 		),
 	}
 
-	cmd.Flags().String("cloud", "", "Specify the cloud provider as aws.")
-	cmd.Flags().String("service", "", "Name of an AWS VPC endpoint service.")
+	cmd.Flags().String("cloud", "", "Specify the cloud provider as aws or azure.")
```

Comment:
Can you use `utils.ArrayToCommaDelimitedString()` here so the options are quoted?

Reply:
yes updated thanks